homeassistant/components/device_tracker/legacy.py

```diff
@@ -649,6 +649,7 @@ def __init__(
         self.consider_home = consider_home
 
         # Device ID
+        self._attr_unique_id = dev_id
```

Comment:
device can be created just by sending a `see` service call, without any implementing integration to it... I thought about adding a unique id there but it doesn't make much sense to add that property to each call (as dev_id is already sent). In https://github.com/home-assistant/core/blob/25b874a6092f463694a12b8de805f4aacc7c5d9e/homeassistant/components/device_tracker/legacy.py#L531 we make sure that the dev_id is unique. I thought we can utilize that, but now, come to think about it, we need to set the unique id to be the entity id, not just the dev id, and then it will be unique. @frenck what do you think?

Reply:
Its documented on our developers portal:

<https://developers.home-assistant.io/docs/core/entity/device-tracker>